change/@fluentui-merge-styles-3328f941-f74b-4a07-8fc0-aca13cacd01f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31703&commitId=c408632b6367a345495de3e5a67473c519b16542&env=prod&clientType=FLUENTUI)


  <details><summary><b>(Dev-Only) Suggestions</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.default.chromium.png|450472| Changed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over a wide suggestion element.chromium.png|0| Removed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over the X button on a wide suggestion element.chromium.png|0| Removed|

</details>


  <details><summary><b>Breadcrumb</b> 33 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Root.default.chromium.png|305097| Changed|
|Breadcrumb.Root - RTL.default.chromium.png|314277| Changed|
|Breadcrumb.Button.default.chromium.png|294539| Changed|
|Breadcrumb.Button - RTL.default.chromium.png|302823| Changed|
|Breadcrumb.Hovering items.chromium.png|1608748| Changed|
|Breadcrumb.Button - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.hover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.click.chromium.png|0| Removed|
|Breadcrumb.Button.hover.chromium.png|0| Removed|
|Breadcrumb.Button.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Button.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Button.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.actionable hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.non-actionable hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.hover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.click.chromium.png|0| Removed|
|Breadcrumb.Root.hover.chromium.png|0| Removed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Root.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Root.shortTitleHover.chromium.png|0| Removed|

</details>


  <details><summary><b>Button (compat)</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).No Icon.default.chromium.png|236698| Changed|
|Button (compat).Disabled.default.chromium.png|237391| Changed|
|Button (compat).Primary Anchor.default.chromium.png|235344| Changed|
|Button (compat).Root.default.chromium.png|237342| Changed|
|Button (compat).Primary Disabled.default.chromium.png|237420| Changed|
|Button (compat).Root - RTL.default.chromium.png|237347| Changed|
|Button (compat).Primary Checked.default.chromium.png|235484| Changed|
|Button (compat).Icon Only.default.chromium.png|236676| Changed|
|Button (compat).Checked.default.chromium.png|237100| Changed|
|Button (compat).Primary.default.chromium.png|235591| Changed|
|Button (compat).Checked.hover.chromium.png|0| Removed|
|Button (compat).Checked.pressed.chromium.png|0| Removed|
|Button (compat).Disabled.hover.chromium.png|0| Removed|
|Button (compat).Disabled.pressed.chromium.png|0| Removed|
|Button (compat).Icon Only.hover.chromium.png|0| Removed|
|Button (compat).Icon Only.pressed.chromium.png|0| Removed|
|Button (compat).No Icon.hover.chromium.png|0| Removed|
|Button (compat).No Icon.pressed.chromium.png|0| Removed|
|Button (compat).Primary Anchor.hover.chromium.png|0| Removed|
|Button (compat).Primary Anchor.pressed.chromium.png|0| Removed|
|Button (compat).Primary Checked.hover.chromium.png|0| Removed|
|Button (compat).Primary Checked.pressed.chromium.png|0| Removed|
|Button (compat).Primary Disabled.hover.chromium.png|0| Removed|
|Button (compat).Primary Disabled.pressed.chromium.png|0| Removed|
|Button (compat).Primary.hover.chromium.png|0| Removed|
|Button (compat).Primary.pressed.chromium.png|0| Removed|
|Button (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button (compat).Root.hover.chromium.png|0| Removed|
|Button (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Action (compat)</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action (compat).Checked.default.chromium.png|237030| Changed|
|Button Action (compat).Disabled.default.chromium.png|237113| Changed|
|Button Action (compat).No Icon.default.chromium.png|236333| Changed|
|Button Action (compat).Root.default.chromium.png|237051| Changed|
|Button Action (compat).Icon Only.default.chromium.png|235598| Changed|
|Button Action (compat).Root - RTL.default.chromium.png|237348| Changed|
|Button Action (compat).Checked.hover.chromium.png|0| Removed|
|Button Action (compat).Checked.pressed.chromium.png|0| Removed|
|Button Action (compat).Disabled.hover.chromium.png|0| Removed|
|Button Action (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Action (compat).Icon Only.hover.chromium.png|0| Removed|
|Button Action (compat).Icon Only.pressed.chromium.png|0| Removed|
|Button Action (compat).No Icon.hover.chromium.png|0| Removed|
|Button Action (compat).No Icon.pressed.chromium.png|0| Removed|
|Button Action (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Action (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Action (compat).Root.hover.chromium.png|0| Removed|
|Button Action (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Command (compat)</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command (compat).Checked.default.chromium.png|252752| Changed|
|Button Command (compat).Disabled.default.chromium.png|253332| Changed|
|Button Command (compat).Root - RTL.default.chromium.png|255582| Changed|
|Button Command (compat).Root.default.chromium.png|253219| Changed|
|Button Command (compat).Checked.hover.chromium.png|0| Removed|
|Button Command (compat).Checked.open.chromium.png|0| Removed|
|Button Command (compat).Checked.pressed.chromium.png|0| Removed|
|Button Command (compat).Disabled.hover.chromium.png|0| Removed|
|Button Command (compat).Disabled.open.chromium.png|0| Removed|
|Button Command (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.open.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Command (compat).Root.hover.chromium.png|0| Removed|
|Button Command (compat).Root.open.chromium.png|0| Removed|
|Button Command (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Compound (compat)</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound (compat).No Icon.default.chromium.png|250672| Changed|
|Button Compound (compat).Checked.default.chromium.png|248914| Changed|
|Button Compound (compat).Disabled.default.chromium.png|250222| Changed|
|Button Compound (compat).Primary Checked.default.chromium.png|238491| Changed|
|Button Compound (compat).Primary Disabled.default.chromium.png|250402| Changed|
|Button Compound (compat).Primary.default.chromium.png|239670| Changed|
|Button Compound (compat).Root - RTL.default.chromium.png|252027| Changed|
|Button Compound (compat).Root.default.chromium.png|250570| Changed|
|Button Compound (compat).Checked.hover.chromium.png|0| Removed|
|Button Compound (compat).Checked.pressed.chromium.png|0| Removed|
|Button Compound (compat).Disabled.hover.chromium.png|0| Removed|
|Button Compound (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Compound (compat).No Icon.hover.chromium.png|0| Removed|
|Button Compound (compat).No Icon.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary Checked.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary Checked.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary Disabled.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary Disabled.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary.pressed.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Compound (compat).Root.hover.chromium.png|0| Removed|
|Button Compound (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Special Scenarios (compat)</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios (compat).no flex shrink.default.chromium.png|247331| Changed|
|Button Special Scenarios (compat).primary with placeholder.default.chromium.png|236425| Changed|

</details>


  <details><summary><b>Button Split (compat)</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Root - RTL.default.chromium.png|259069| Changed|
|Button Split (compat).Checked.default.chromium.png|255602| Changed|
|Button Split (compat).Command Split.default.chromium.png|252805| Changed|
|Button Split (compat).Primary Checked.default.chromium.png|252669| Changed|
|Button Split (compat).Primary Disabled.default.chromium.png|256092| Changed|
|Button Split (compat).Root.default.chromium.png|256072| Changed|
|Button Split (compat).Primary with Primary Action Disabled.default.chromium.png|255532| Changed|
|Button Split (compat).Default with Primary Action Disabled.default.chromium.png|256135| Changed|
|Button Split (compat).Primary.default.chromium.png|252881| Changed|
|Button Split (compat).Disabled.default.chromium.png|256133| Changed|
|Button Split (compat).Checked.hover main.chromium.png|0| Removed|
|Button Split (compat).Checked.hover split.chromium.png|0| Removed|
|Button Split (compat).Command Split.hover main.chromium.png|0| Removed|
|Button Split (compat).Command Split.hover split.chromium.png|0| Removed|
|Button Split (compat).Default with Primary Action Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Default with Primary Action Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary Checked.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary Checked.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary with Primary Action Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary with Primary Action Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary.hover split.chromium.png|0| Removed|
|Button Split (compat).Root - RTL.hover main.chromium.png|0| Removed|
|Button Split (compat).Root - RTL.hover split.chromium.png|0| Removed|
|Button Split (compat).Root.hover main.chromium.png|0| Removed|
|Button Split (compat).Root.hover split.chromium.png|0| Removed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|1608609| Changed|

</details>


  <details><summary><b>ComboBox</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Error.default.chromium.png|280882| Changed|
|ComboBox.Disabled.default.chromium.png|275816| Changed|
|ComboBox.Root.default.chromium.png|275538| Changed|
|ComboBox.Placeholder.default.chromium.png|275544| Changed|
|ComboBox.Root - RTL.default.chromium.png|281697| Changed|
|ComboBox.Styled.default.chromium.png|275556| Changed|
|ComboBox.Disabled.click.chromium.png|0| Removed|
|ComboBox.Disabled.hover.chromium.png|0| Removed|
|ComboBox.Error.click.chromium.png|0| Removed|
|ComboBox.Error.hover.chromium.png|0| Removed|
|ComboBox.Placeholder.click.chromium.png|0| Removed|
|ComboBox.Placeholder.hover.chromium.png|0| Removed|
|ComboBox.Root - RTL.click.chromium.png|0| Removed|
|ComboBox.Root - RTL.hover.chromium.png|0| Removed|
|ComboBox.Root.click.chromium.png|0| Removed|
|ComboBox.Root.hover.chromium.png|0| Removed|
|ComboBox.Styled.click.chromium.png|0| Removed|
|ComboBox.Styled.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>CommandBar</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Root.default.chromium.png|324752| Changed|
|CommandBar.Icons only.default.chromium.png|280882| Changed|
|CommandBar.Root - RTL.default.chromium.png|335452| Changed|
|CommandBar.Text only.default.chromium.png|300058| Changed|
|CommandBar.Icons only.click.chromium.png|0| Removed|
|CommandBar.Icons only.hover.chromium.png|0| Removed|
|CommandBar.Root - RTL.click.chromium.png|0| Removed|
|CommandBar.Root - RTL.hover.chromium.png|0| Removed|
|CommandBar.Root.click.chromium.png|0| Removed|
|CommandBar.Root.hover.chromium.png|0| Removed|
|CommandBar.Text only.click.chromium.png|0| Removed|
|CommandBar.Text only.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ContextualMenu</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With submenus with hrefs.chromium.png|1608740| Changed|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|0| Removed|

</details>


  <details><summary><b>Dialog</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Blocking.default.chromium.png|278615| Changed|
|Dialog.Wide Dialog.default.chromium.png|309911| Changed|
|Dialog.Root.default.chromium.png|278641| Changed|
|Dialog.Root - RTL.default.chromium.png|285948| Changed|
|Dialog.Large header.default.chromium.png|279294| Changed|

</details>


  <details><summary><b>Dropdown</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.hover item.chromium.png|0| Removed|
|Dropdown.Disabled option selected.click.chromium.png|0| Removed|
|Dropdown.Disabled option selected.hover item.chromium.png|0| Removed|
|Dropdown.Required without label.click.chromium.png|0| Removed|
|Dropdown.Required without label.hover item.chromium.png|0| Removed|
|Dropdown.Required.click.chromium.png|0| Removed|
|Dropdown.Required.hover item.chromium.png|0| Removed|
|Dropdown.Root - RTL.click.chromium.png|0| Removed|
|Dropdown.Root - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Root.click.chromium.png|0| Removed|
|Dropdown.Root.hover item.chromium.png|0| Removed|

</details>


  <details><summary><b>Facepile</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Facepile.Add face.default.chromium.png|238154| Changed|
|Facepile.Custom button styles.default.chromium.png|237597| Changed|
|Facepile.Overflow.default.chromium.png|237421| Changed|

</details>


  <details><summary><b>FocusTrapZones</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|904630| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|904715| Changed|
|FocusTrapZones.Panel on its own.default.chromium.png|1605919| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|0| Removed|

</details>


  <details><summary><b>IconButton Scenarios (compat)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|IconButton Scenarios (compat).normal icon button.icon.chromium.png|235349| Changed|
|IconButton Scenarios (compat).icon button with menu.icon.chromium.png|235924| Changed|
|IconButton Scenarios (compat).icon button with menu.hover icon.chromium.png|0| Removed|
|IconButton Scenarios (compat).normal icon button.hover icon.chromium.png|0| Removed|

</details>


  <details><summary><b>MessageBar</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Root dismiss - RTL.default.chromium.png|242430| Changed|
|MessageBar.Root actions single line.default.chromium.png|246744| Changed|
|MessageBar.Root actions.default.chromium.png|246430| Changed|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|247179| Changed|
|MessageBar.Root dismiss and action single line.default.chromium.png|247173| Changed|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|249257| Changed|
|MessageBar.Root dismiss single line.default.chromium.png|242424| Changed|
|MessageBar.Root actions single line - RTL.default.chromium.png|246750| Changed|
|MessageBar.Root dismiss and action.default.chromium.png|247655| Changed|
|MessageBar.Root actions - RTL.default.chromium.png|247872| Changed|
|MessageBar.Root dismiss.default.chromium.png|242424| Changed|
|MessageBar.Root truncated.default.chromium.png|283467| Changed|
|MessageBar.Root truncated - RTL.default.chromium.png|283461| Changed|

</details>


  <details><summary><b>Nav</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Nav.Disabled.default.chromium.png|278356| Changed|
|Nav.Disabled - RTL.default.chromium.png|285123| Changed|
|Nav.Root - RTL.default.chromium.png|306911| Changed|
|Nav.Root.default.chromium.png|299642| Changed|
|Nav.Disabled - RTL.click.chromium.png|0| Removed|
|Nav.Disabled - RTL.hover.chromium.png|0| Removed|
|Nav.Disabled.click.chromium.png|0| Removed|
|Nav.Disabled.hover.chromium.png|0| Removed|
|Nav.Root - RTL.click.chromium.png|0| Removed|
|Nav.Root - RTL.hover.chromium.png|0| Removed|
|Nav.Root.click.chromium.png|0| Removed|
|Nav.Root.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>OverflowSet</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root - RTL.default.chromium.png|238339| Changed|
|OverflowSet.Root.default.chromium.png|238336| Changed|
|OverflowSet.Root - RTL.default.chromium_1.png|0| Removed|
|OverflowSet.Root.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>OverflowSet variant</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet variant.Vertical Direction.default.chromium.png|237165| Changed|

</details>


  <details><summary><b>Panel</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.Custom anchored left.default.chromium.png|1606165| Changed|
|Panel.Large fixed right.default.chromium.png|1607333| Changed|
|Panel.Custom.default.chromium.png|1609012| Changed|
|Panel.Extra large right.default.chromium.png|1607476| Changed|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|1606048| Changed|
|Panel.Large right.default.chromium.png|1607531| Changed|
|Panel.Medium right.default.chromium.png|1607411| Changed|
|Panel.Small fluid right.default.chromium.png|1608879| Changed|
|Panel.Medium right - RTL.default.chromium.png|1607412| Changed|
|Panel.Small fixed right w- close button.default.chromium.png|1606048| Changed|
|Panel.SearchBox and Right Panel.default.chromium.png|1610294| Changed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|1610294| Changed|
|Panel.Small left w- close button - RTL.default.chromium.png|1606314| Changed|
|Panel.With footer at the bottom.default.chromium.png|1605405| Changed|
|Panel.Small left w- close button.default.chromium.png|1606361| Changed|
|Panel.With custom navigation.default.chromium.png|1605627| Changed|
|Panel.With no header, close button.default.chromium.png|1606450| Changed|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel.click.chromium.png|0| Removed|

</details>


  <details><summary><b>PeoplePicker</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Compact selected - RTL.default.chromium.png|243441| Changed|
|PeoplePicker.List selected - RTL.default.chromium.png|242790| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|246273| Changed|
|PeoplePicker.Compact selected.default.chromium.png|243454| Changed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|1603867| Changed|
|PeoplePicker.List selected.default.chromium.png|241961| Changed|
|PeoplePicker.Normal selected.default.chromium.png|243454| Changed|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|247655| Changed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|243441| Changed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.List.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal.suggestions.chromium.png|0| Removed|

</details>


  <details><summary><b>Pivot</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Count and icon.default.chromium.png|283672| Changed|
|Pivot.Large.default.chromium.png|258446| Changed|
|Pivot.Count and icon - RTL.default.chromium.png|286701| Changed|
|Pivot.Root.default.chromium.png|254422| Changed|
|Pivot.Tabs.default.chromium.png|245848| Changed|
|Pivot.Tabs large.default.chromium.png|247516| Changed|
|Pivot.Tabs - RTL.default.chromium.png|247365| Changed|
|Pivot.Count and icon - RTL.click.chromium.png|0| Removed|
|Pivot.Count and icon - RTL.hover-selected.chromium.png|0| Removed|
|Pivot.Count and icon - RTL.hover.chromium.png|0| Removed|
|Pivot.Count and icon.click.chromium.png|0| Removed|
|Pivot.Count and icon.hover-selected.chromium.png|0| Removed|
|Pivot.Count and icon.hover.chromium.png|0| Removed|
|Pivot.Large.click.chromium.png|0| Removed|
|Pivot.Large.hover-selected.chromium.png|0| Removed|
|Pivot.Large.hover.chromium.png|0| Removed|
|Pivot.Root.click.chromium.png|0| Removed|
|Pivot.Root.hover-selected.chromium.png|0| Removed|
|Pivot.Root.hover.chromium.png|0| Removed|
|Pivot.Tabs - RTL.click.chromium.png|0| Removed|
|Pivot.Tabs - RTL.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs - RTL.hover.chromium.png|0| Removed|
|Pivot.Tabs large.click.chromium.png|0| Removed|
|Pivot.Tabs large.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs large.hover.chromium.png|0| Removed|
|Pivot.Tabs.click.chromium.png|0| Removed|
|Pivot.Tabs.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Pivot - Overflow</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.chromium.png|1606542| Changed|
|Pivot - Overflow.Root.chromium.png|1608138| Changed|
|Pivot - Overflow.Root.Medium.chromium.png|0| Removed|
|Pivot - Overflow.Root.Wide.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Medium.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Wide.chromium.png|0| Removed|

</details>


  <details><summary><b>ResizeGroup</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ResizeGroup.Root.default.chromium.png|253657| Changed|
|ResizeGroup.Root - RTL.default.chromium.png|253663| Changed|
|ResizeGroup.Root - RTL.click overflow.chromium.png|0| Removed|
|ResizeGroup.Root.click overflow.chromium.png|0| Removed|

</details>


  <details><summary><b>SearchBox</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.ShowIcon.default.chromium.png|241160| Changed|
|SearchBox.ShowIcon - RTL.default.chromium.png|241166| Changed|
|SearchBox.ShowIcon - RTL.default.chromium_1.png|0| Removed|
|SearchBox.ShowIcon.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>SpinButton</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton.Label at end with icon.default.chromium.png|242601| Changed|
|SpinButton.Label at end.default.chromium.png|242623| Changed|
|SpinButton.Label on bottom with icon.default.chromium.png|247695| Changed|
|SpinButton.Label at end with icon - RTL.default.chromium.png|242614| Changed|
|SpinButton.Disabled.default.chromium.png|242785| Changed|
|SpinButton.With icon - RTL.default.chromium.png|242607| Changed|
|SpinButton.Label on top with icon.default.chromium.png|251269| Changed|
|SpinButton.Label at end - RTL.default.chromium.png|242636| Changed|
|SpinButton.With icon.default.chromium.png|242603| Changed|
|SpinButton.Label on top.default.chromium.png|249887| Changed|
|SpinButton.Label on bottom.default.chromium.png|247697| Changed|
|SpinButton.Root.default.chromium.png|242630| Changed|
|SpinButton.Disabled.click input.chromium.png|0| Removed|
|SpinButton.Disabled.hover arrow.chromium.png|0| Removed|
|SpinButton.Disabled.hover input.chromium.png|0| Removed|
|SpinButton.Disabled.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.Root.click input.chromium.png|0| Removed|
|SpinButton.Root.hover arrow.chromium.png|0| Removed|
|SpinButton.Root.hover input.chromium.png|0| Removed|
|SpinButton.Root.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.With icon - RTL.click input.chromium.png|0| Removed|
|SpinButton.With icon - RTL.hover arrow.chromium.png|0| Removed|
|SpinButton.With icon - RTL.hover input.chromium.png|0| Removed|
|SpinButton.With icon - RTL.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.With icon.click input.chromium.png|0| Removed|
|SpinButton.With icon.hover arrow.chromium.png|0| Removed|
|SpinButton.With icon.hover input.chromium.png|0| Removed|
|SpinButton.With icon.mouseDown arrow.chromium.png|0| Removed|

</details>


  <details><summary><b>Sticky breadcrumb and sticky details list header</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down by a small amount so that the first row is still visible and the header becomes sticky.chromium.png|760656| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll back to the top when horizontal scroll is non-zero.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down to the bottom.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down when horizontal scroll is non-zero.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll right.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll up to the top.chromium.png|0| Removed|

</details>


  <details><summary><b>TagItem</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagItem.Selected.default.chromium.png|241874| Changed|
|TagItem.Selected - RTL.default.chromium.png|241880| Changed|
|TagItem.Selected - RTL.Tag Item Clear Button Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Select.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Clear Button Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Clear Button Selected Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Select.chromium.png|0| Removed|

</details>


  <details><summary><b>TagPicker</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Selected.default.chromium.png|241874| Changed|
|TagPicker.With long tag.default.chromium.png|247570| Changed|
|TagPicker.Selected - RTL.default.chromium.png|241880| Changed|
|TagPicker.Root.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|0| Removed|

</details>


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.SmallHeadline.default.chromium.png|242702| Changed|
|TeachingBubble.WideIllustration.default.chromium.png|262741| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|270417| Changed|

</details>


  <details><summary><b>ThemeProvider</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider.Customized styles.default.chromium.png|236321| Changed|
|ThemeProvider.Customized theme.default.chromium.png|236421| Changed|
|ThemeProvider.Customized theme - nested ThemeProvider.default.chromium.png|239838| Changed|
|ThemeProvider.Default theme.default.chromium.png|235871| Changed|

</details>


  <details><summary><b>ThemeProvider with Customizer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with Customizer.Customizer wraps ThemeProvider.default.chromium.png|241177| Changed|
|ThemeProvider with Customizer.ThemeProvider wraps Customizer.default.chromium.png|243295| Changed|

</details>


  <details><summary><b>ThemeProvider with loadTheme</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.default.chromium.png|244206| Changed|
|ThemeProvider with loadTheme.Use updated global theme.default.chromium.png|236666| Changed|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.global theme changed.chromium.png|0| Removed|
|ThemeProvider with loadTheme.Use updated global theme.global theme changed.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>